index.html

```diff
@@ -82,37 +88,36 @@ <h2 id="modalTitle">Add New Item</h2>
     </div>
     </div>
 
+    <!-- FILTERS AND OPTIONS SIDEBAR -->
+     <div id="optionsModal" class="optionsModal" role="dialog" aria-labelledby="optionsModalTitle" aria-modal="true" >
+       <div class="optionsSettingsContainer">
+        <div class="optionsModalTitleContainer">
+        <h2 class="optionsModalTitle">Settings</h2>
+        <button id="closeOptionsModal" class="close" aria-label="Close Settings Panel">&times;</button>
+        </div>
+       <div class="filterInputs">
+        <label class="itemsControlInput">
+          <input type="checkbox" id="autoSort" class="checkbox" />
+          Auto Sort
+          </label>
+          <label class="itemsControlInput">
+            <input type="checkbox" id="hideChecked" class="checkbox" />
+          Hide Checked
+          </label>
+       </div>
```

Comment:
All of these options should be tied to their own table in the database, I think. 

It's handy to set up the view how you like it, close it, move on, and not have to reset it if the page if refreshed. 

This won't be done here, but in a future PR